### PR TITLE
CTCTRADERS-505: Refactor Trader to only be a single model

### PR DESCRIPTION
**Pull request by MatthewAlexanderBeck**

### CODE REVIEW

File: test/models/messages/TraderSpec.scala [hunk 1]
```diff
@@ -14,247 +14,86 @@
  * limitations under the License.
  */
 
+///*
```
Comment (chris-ianson) [inline test/models/messages/TraderSpec.scala:17]: Remove comments

Comment (MatthewAlexanderBeck) [reply test/models/messages/TraderSpec.scala:17]: Double the disclaimer, double the fun

File: app/services/conversion/ArrivalNotificationConversionService.scala [hunk 2]
```diff
@@ -99,13 +99,13 @@ class ArrivalNotificationConversionService {
...
-      name            = Some(traderName),
-      streetAndNumber = Some(traderAddress.buildingAndStreet),
-      postCode        = Some(traderAddress.postcode),
-      city            = Some(traderAddress.city),
-      countryCode     = Some(countryCode_GB)
+      name            = traderName,
+      streetAndNumber = traderAddress.buildingAndStreet,
+      postCode        = traderAddress.postcode,
+      city            = traderAddress.city,
+      countryCode     = countryCode_GB
```
Comment (amaalali) [inline app/services/conversion/ArrivalNotificationConversionService.scala:109]: This doesn't seem to be needed anymore? We can just in-line this at the call site?

File: app/services/conversion/SubmissionModelService.scala [hunk 3]
```diff
@@ -58,26 +58,15 @@ class SubmissionModelService @Inject()() {
...
-  }
+  private def buildTrader(trader: Trader): Trader =
+    Trader(
+      name            = trader.name,
+      streetAndNumber = trader.streetAndNumber,
+      postCode        = trader.postCode,
+      city            = trader.city,
+      countryCode     = trader.countryCode,
+      eori            = trader.eori
+    )
```
Comment (amaalali) [inline app/services/conversion/SubmissionModelService.scala:69]: I don't think this is needed anymore?

**Pull request merged**